Emby.Server.Implementations/ApplicationHost.cs

```diff
@@ -1408,7 +1408,7 @@ public async Task<SystemInfo> GetSystemInfo(CancellationToken cancellationToken)
                 InternalMetadataPath = ApplicationPaths.InternalMetadataPath,
                 CachePath = ApplicationPaths.CachePath,
                 HttpServerPortNumber = HttpPort,
-                SupportsHttps = SupportsHttps,
+                SupportsHttps = CanConnectWithHttps,
                 HttpsPortNumber = HttpsPort,
```

Comment:
Can we get rid of the `HttpServerPortNumber`, `SupportsHttps` and `HttpsPortNumber` properties here? A quick scan of jellyfin-web shows that these don't appear to be used, and I don't really know how a client would use these properties anyways. In fact it's a bit of a security vulnerability to be reporting the internal port numbers if the server is behind a reverse proxy.

Additionally the `CanConnectWithHttps` property doesn't even report anything useful here if the server is behind a reverse proxy because `PublicPort` and `PublicHttpsPort` are not considered here.

Would like some feedback on this before I move forward with it because I'm not familiar enough with how all the clients work to know if I'm missing something

Reply:
I would say rip them out and fix clients which break - they have no use for internal ports anyway (internal ports should not be accessible).

Reply:
Yup, already done in my latest commits :) Sorry, forgot to resolve this thread